articles/how-artificially-intelligent-chatbots-have-impacted-healthcare-industry/index.md

```diff
@@ -0,0 +1,59 @@
+### Introduction
+In recent times, organizations have adopted the benefits offered by AI-powered [virtual assistants](https://searchcustomerexperience.techtarget.com/definition/virtual-assistant-AI-assistant) to automate routine tasks and give their clients attention in customer services, payments, and marketing.
+
+The healthcare industry has now adopted this trend. [AI](https://www.ibm.com/cloud/learn/what-is-artificial-intelligence) has created an environment where the patient outcome has been improved with minimum costs involved.
```

Comment:
AI creates an environment suitable to improve the patient's health with minimal costs and overheads.